global.json

```diff
@@ -8,7 +8,7 @@
     }
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "5.0.0-beta.20280.1",
-    "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.20280.1"
+    "Microsoft.DotNet.Arcade.Sdk": "5.0.0-beta.20309.1",
+    "Microsoft.DotNet.Helix.Sdk": "5.0.0-beta.20311.2"
```

Comment:
Needed to get a version that can run > 1 work item per machine and support "package as payload" mode.

Reply:
Could you share some details how is that implemented and what was the change?

Reply:
1) Your fix to allow the nupkg to be a payload == "support 'package as payload mode' " and 2) to fix you quitting as soon as it's already installed == can run > 1 work item per machine